src/VirtoCommerce.CatalogModule.Core/Model/Asset.cs

```diff
@@ -19,6 +21,21 @@ public string ReadableSize
 
         public byte[] BinaryData { get; set; }
 
+        public override IList<SeoInfo> SeoInfos
```

Comment:
You don't need to create a SeoInfo with empty values. Move this logic into the  base class and populate specific fields in the derived. 
You can use RelativeURL as slug

Reply:
fixed